### PR TITLE
[new release] batch_jaro_winkler (0.1.1)

### PR DESCRIPTION
**Pull request by dbousque**

### CODE REVIEW

File: packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+homepage: "https://github.com/dbousque/batch_jaro_winkler"
+dev-repo: "git://github.com/dbousque/batch_jaro_winkler.git"
+bug-reports: "https://github.com/dbousque/batch_jaro_winkler/issues"
+synopsis: "Fast batch jaro winkler distance implementation in C99"
+description: """
+This project gets its performance from the pre-calculation of an optimized
+model in advance of the actual runtime calculations. Supports any encoding.
+"""
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
```
Comment (mseri) [inline packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs "--root=./ocaml/lib"]
```

Comment (mseri) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: I think with your current structure you need to tell dune where to look for the sources

File: packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam [hunk 2]
```diff
@@ -11,7 +11,7 @@ This project gets its performance from the pre-calculation of an optimized
 model in advance of the actual runtime calculations. Supports any encoding.
 """
 build: [
-  ["dune" "build" "-p" name "-j" jobs "--root=./ocaml/lib"]
+  ["dune" "build" "-j" jobs "--root=./ocaml/lib"]
```
Comment (mseri) [inline packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: Keep the p, it includes also the release flag and target the specific named package only

Comment (mseri) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: You can ignore the warning in the lint step

Comment (dbousque) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: Hi there, thanks for the feedback. Sadly, `-p` and `--root` seem to not be compatible. See here: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/782baf0b2e7235b4486e5c6e72286f0c4db38f17/variant/compilers,4.02,batch_jaro_winkler.0.1.1. `# dune: Cannot use --root and -p simultaneously`. I have a hard time getting the linter to find the `dune-project` file: `The package seems to use dune but the dune-project file is missing.`. I tried to include it at the root, but then other things fail. Do you have an idea of what I could do?

Comment (dbousque) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: Sorry I didn't see your message about the lint step before. Nice! So do you prefer using `-p` or `--root`?

Comment (mseri) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam:14]: Then do what `-p` does (https://dune.readthedocs.io/en/stable/opam.html) adding
`--only-packages pkg --profile release --default-target @install`
that line, or using `-p` and building in the right folder with [ "cd" "ocaml/lib"] before the dune invocation

Comment (dbousque) [reply packages/batch_jaro_winkler/batch_jaro_winkler.0.1.1/opam]: Thanks! It finally passed when using your suggestion, except for the lint.

**Comment by mseri** [comment]
Looks fine now, thanks!

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum. We have an announcement tag and a community category for this purpose

**Pull request merged**